iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/pipe/connector/payload/request/TransferConfigPlanReq.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iotdb.commons.pipe.connector.payload.request;
+
+import org.apache.iotdb.service.rpc.thrift.TPipeTransferReq;
+
+public abstract class TransferConfigPlanReq extends TPipeTransferReq {}
```

Comment:
May better not use this abstract class, since no class else will need this. Make configNode's PipeTransferConfigPlanReq directly extend TPipeTransferReq.

Reply:
PipeTransferConfigPlanReq is in CN package, so receiver on DN won't be able to directly import it. This abstract class is intended to be imported in the receiver.